index.html

```diff
@@ -0,0 +1,137 @@
+    <title>Home CC Bootcamp</title>
+    <link href="css/main.css" rel="stylesheet">
+    <script src="scripts/main.js" defer></script>
+</head>
+<body class="mainContainer">
+    <header class="header">
+        <nav class="navbar navbar-black social desktop desktop-flex">
+            <ul class="nav nav-start">
+                <li>
+                    <a rel="noopener noreferrer" href="https://github.com/JohnFTitor" target="_blank" class="social-icon"><img src="assets/icons/github.svg" alt="Github Link"></a>       
+                </li>   
+                <li>
+                    <a rel="noopener noreferrer" href="https://twitter.com/johnftitor" target="_blank" class="social-icon"><img src="assets/icons/twitter.svg" alt="Twitter Link"></a>       
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">Spanish</a>   
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">My Page</a>   
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">Log Out</a>   
+                </li> 
+            </ul>
+        </nav>
+        <nav class="navbar">
+            <ul class="nav">
+                <li class="mobile"><button type="button" id="menu"><img src="assets/icons/menu.svg" alt="Menu Button"></button></li>
+                <li class="desktop"><a rel="noopener noreferrer" href="index.html"><img src="assets/images/logo.svg" alt="Home Page"></a></li>
+                <li class="desktop">
+                    <ul class="nav nav-links">
+                        <li >
+                            <a rel="noopener noreferrer" href="about.html"> About </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="index.html#main-program"> Program </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> Join </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> Sponsor </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> News </a>
+                        </li>
+                        <li>
+                            <button type="button" class="interaction interaction-alternative">CC Campaing</button>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="content">
+        <section id="presentation">
+            <div class="container presentation">
+                <p class="title title-description title-description-home">"Welcome! Future Developer"</p>
+                <h1 class="title title-header title-home">CREATIVE COMMONS BOOTCAMP SUMMER 2022</h1>
```

Comment:
- [ ] Here, I advise you maintain one `h1` element per page for the same reason as highlighted above:point_up: